README.md

```diff
@@ -11,6 +11,30 @@
 - [Run a SignalFlow computation](#run-a-signalflow-computation)
 - [License](#license)
 
+SignalFlow is the SignalFx real-time analytics computation language. The SignalFlow
+API allows SignalFx users to execute real-time streaming analytics computations on
+the SignalFx platform. For more information, see the Splunk Observability Cloud
+developer documentation:
+
+* [SignalFlow Overview](https://dev.splunk.com/observability/docs/signalflow/)
+* [SignalFlow API Reference](https://dev.splunk.com/observability/reference/api/signalflow/latest)
+
+The SignalFlow Python library is a client you can use to send SignalFlow programs
```

Comment:
Instead of "send signalflow programs to the backend" could be something more like "execute signalflow queries against the backend and stream data back to the client".